contracts/scripts/server-add-erc20.ts

```diff
@@ -55,6 +112,32 @@ async function main() {
             console.log(JSON.stringify(await addToken(token), null, 2));
         });
 
+    program
+        .command('add-and-mint')
+        .option('-sn, --sname <s_name>')
+        .option('-s, --symbol <symbol>')
+        .option('-d, --decimals <decimals>')
+        .option('-e, --env <env>')
+        .action(async (cmd: Command) => {
+            const token: Token = {
```

Comment:
prefer deconstruction, such as:
```suggestion
        .action(async ({ decimals, s_name, address, symbol }: Command) => {
            const token: Token = {
```

Reply:
Great, just added